core/components/atoms/collapsible/__stories__/index.story.jsx

```diff
@@ -23,7 +23,7 @@ export const all = () => {
 const customCode = `() => {
   const [expanded, setExpanded] = React.useState(false);
   return (
-    <div style={{height: "100vh"}}>
+    <div className="d-flex">
```

Comment:
Please verify and remove the wrapper div as it does not seem to be affecting anything.